core/scripts/functions/src/generate_jobspecs_cmd.go

```diff
@@ -31,6 +31,8 @@ func (g *generateJobSpecs) Run(args []string) {
 	donID := fs.String("donid", "", "don id string")
 	routerAddress := fs.String("contract", "", "router contract address")
 	truncateHostname := fs.Bool("truncateboothostname", false, "truncate host name to first segment (needed for staging DONs)")
+	gatewayID := fs.String("gatewayid", "", "gateway id string")
+	gatewayURL := fs.String("gatewayurl", "", "gateway url string")
 	err := fs.Parse(args)
 	if err != nil || nodesFile == nil || *nodesFile == "" || routerAddress == nil || *routerAddress == "" {
```

Comment:
Is it actually a required field?
I was going off of since we aren't checking things like donID.. but looks like that might just be out of date and we _should_ require that.

Reply:
Yeah I think everything that is in latest oracle.toml should be required.